Gemfile.lock

```diff
@@ -56,13 +56,30 @@ GEM
       minitest (~> 5.1)
```

Comment:
Looks like there are a lot of new packages in here that look like they're coming out of no-where. Is stuff like `addressable` and `public-suffix` needed?

Reply:
There are definitely alot of useless gems in this file. I think they were added when I installed capybara (before I checked with you if we needed capybara). I've removed everything that was added for capybara.